src/api/eth.rs

```diff
@@ -136,6 +136,24 @@ impl<T: Transport> Eth<T> {
         CallFuture::new(result)
     }
 
+    /// Get block details with full transaction objects.
+    pub fn block_with_raw_txs(&self, block: BlockId) -> CallFuture<Option<Block<RawTransactionDetails>>, T::Out> {
```

Comment:
I guess the are only few differences between `RawTransactionDetails` and `Transaction`:
- `from` being `Optional`
- `r,s,v` for signature details.

Given this, I think I'd rather be for unifying the two structs and leaving the fields `Optional`, what do you think?

Reply:
@vorot93 do you mind rebasing and merging `RawTransactionDetails` and `Transaction` by having all the fields as `Option`?

Reply:
done